README.md

```diff
@@ -1,5 +1,7 @@
 BitShares Core
 ==============
+[![Build Status](https://travis-ci.org/bitshares/bitshares-core.svg?branch=develop)](https://travis-ci.org/bitshares/bitshares-core)
```

Comment:
Show three icons: master, develop, hardfork.

Reply:
what about the `testnet` branch ?